libs/blocks/merch/merch.js

```diff
@@ -452,6 +452,7 @@ export async function buildCta(el, params) {
   const cta = service.createCheckoutLink(context, text);
   if (el.href.includes('#_tcl')) {
     el.href = el.href.replace('#_tcl', '');
+    cta.classList.add('text-link');
```

Comment:
this should not be necessary with the above.

Reply:
Indeed